ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py

```diff
@@ -110,3 +110,53 @@ def test_numpy_subtract(
         subok=True,
         test_values=False,
     )
+
+
+# square
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_np.valid_float_dtypes),
```

Comment:
We should test all numeric types not just float as this function accepts int dtypes too